### DISCUSSION TITLE
CFGMgmtCamp 2025

### DISCUSSION THREAD

**Discussion by meaksh** [uyuni-project/uyuni#9474] [Show and tell]
# info
* https://cfgmgmtcamp.org/
* Event dates: 3,4,5 February 2025
* CfP: Deadline November 15th, 2024, at 23:59 CET
* https://cfp.cfgmgmtcamp.org/ghent2025/cfp

# types of sessions
- Ignite Talks: 5-minute, auto-forwarding talk (first and second day)
- Presentations: a 45-minute talk on a topic of your choice, with some time (5) for questions. (first and second day)
- Presentations: a 25-minute talk on a topic of your choice, with some time for questions. (first and second day)
- Workshop : 4 to 8 hours of workshop (3rd day only)
- Fringe / Summit : 4 to 8 hours of custom event (3rd day only)

# requirements for the CfP
* Title
* Session type
* Track
* Abstract
* Descripcion (optional)
* Notes (optional)
* Session image (optional)
* Additional speaker (optional)

# attendees
* (add yourself :D)

# Slides

**Comment by RenownTwiz** [comment]
This looks very interesting, unfortunately I don't think I can swing a trip to Europe to attend.  Do they do anything online for remote attendees?

**Reply by meaksh** [reply]
AFAIK in the past they did a streaming of the main track: https://www.youtube.com/watch?v=Ba0TkvPnq-U
